zzz_generated/app-common/app-common/zap-generated/cluster-objects.h

```diff
@@ -2270,6 +2446,62 @@ namespace ShadeConfiguration {
 
 } // namespace ShadeConfiguration
 namespace DoorLock {
+// Enum for DoorLockOperationEventCode
+enum class DoorLockOperationEventCode : uint8_t
+{
+    DOOR_LOCK_OPERATION_EVENT_CODE_UNKNOWN_OR_MFG_SPECIFIC  = 0x00,
```

Comment:
Not really part of this PR, but why do we have that "DOOR_LOCK_OPERATION_EVENT_CODE_" prefix that just duplicates the enum class name?  Followup to drop all those, please.

Reply:
In xml, we have enum

connectedhomeip/src/app/zap-templates/zcl/data-model/chip/wifi-network-diagnostics-cluster.xml

Line 30 in a9ce320

 <item name="802.11a" value="0x00"/> 
without putting the prefix for those enum, it would complain the compilation error.

20_PERCENTER_DIM_UP_IN_0P5_SECONDS_THEN_FADE_TO_OFF_IN_1_SECOND = 0x00,
^
../../zzz_generated/app-common/app-common/zap-generated/cluster-objects.h:1314:5: error: expected identifier
802__11A = 0x00,
^
../../zzz_generated/app-common/app-common/zap-generated/cluster-objects.h:1315:5: error: expected identifier
802__11B = 0x01,
^
../../zzz_generated/app-common/app-common/zap-generated/cluster-objects.h:1316:5: error: expected identifier
802__11G = 0x02,

We need to follow up this. thanks

Reply:
https://github.com/project-chip/connectedhomeip/issues/10261